### DISCUSSION TITLE
How many teams can a developer account be added to?

### DISCUSSION THREAD

**Discussion by dinsen** [fastlane/fastlane#17647] [Q&A] [answered]
We are managing 150+ developer accounts. All with different Apple ID's. 
We want to add one email to all developer accounts so our QA teams don't have to sign in/out every time they need to test an app.
Have anyone experienced a limit of how many teams an Apple ID can be added to?

**Comment by max-ott** [comment, marked as answer]
Previously there was a limit of 50 teams per account, but this seems no longer to be the case.

**Reply by dinsen** [reply]
Hi @max-ott 
Do you have any numbers on how many teams you have on one account?

**Reply by max-ott** [reply]
North of 150 🤐

**Reply by dinsen** [reply]
Thank you @max-ott !

**Reply by jsalazac** [reply]
Hi @Dinsen and @max-ott 
The limit is 500.

**Reply by dinsen** [reply]
Wow 500, that will be more than enough for us 👍

**Reply by rogerluan** [reply]
@jsalazac nice! Is that empirical data, or is it documented somewhere? 🤓
